### PR TITLE
Updated

### PR DESCRIPTION
**Pull request by TylerCousins**
Added our new site!

### CODE REVIEW

**Comment by doomlerd** [comment]
okay nevermind, change it back to v1.0.3, it seems to have been overwritten by the bot.

**Comment by TylerCousins** [comment]
> okay nevermind, change it back to v1.0.3, it seems to have been overwritten by the bot.

ok lol

File: Complex Radio/dist/metadata.json [hunk 1]
```diff
@@ -7,12 +7,12 @@
...
-    "version": "1.0.1",
+    "url": ["complexr.pw", "complexradio.com"],
+    "version": "1.0.3",
     "logo": "https://i.imgur.com/r633Sz0.png",
     "thumbnail": "https://i.imgur.com/1Iey1co.png",
     "color": "#4a9aa8",
     "tags": ["music","radio","gamers","community","complex","complexradio"],
     "category": "music",
-    "regExp": "(([a-z0-9])+[.])?complexr[.]pw"
+    "regExp": "(([a-z0-9])+[.])?complexradio[.]com"
```
Comment (Bas950) [inline Complex Radio/dist/metadata.json:17]: the regex will override the url probs so I doubt complexr.pw will still show the presence, please check if it still works on it or not, if not add it to the regex.

Comment (TylerCousins) [reply Complex Radio/dist/metadata.json:17]: Ok ill have a look

Comment (OfficialCRUGG) [reply Complex Radio/dist/metadata.json:17]: If you want to support both, change the RegExp to `(([a-z0-9])+[.])?(complexradio[.]com|complexr[.]pw)` @TylerCousins

**Comment by TylerCousins** [comment]
> Fix the merge conflicts

I dont get what the conflict is ...

**Pull request closed without merging**